src/app/pages/system/bootenv/bootenv-rename/bootenv-rename.component.ts

```diff
@@ -5,14 +5,14 @@ import { Observable } from 'rxjs';
 import { helptext_system_bootenv } from 'app/helptext/system/boot-env';
 import { FormConfiguration } from 'app/interfaces/entity-form.interface';
 import { EntityFormComponent } from 'app/pages/common/entity/entity-form';
-import { FieldConfig } from 'app/pages/common/entity/entity-form/models/field-config.interface';
+import { FieldSet } from 'app/pages/common/entity/entity-form/models/fieldset.interface';
 import { regexValidator } from 'app/pages/common/entity/entity-form/validators/regex-validation';
 import { BootEnvService, WebSocketService } from 'app/services';
 
 @UntilDestroy()
 @Component({
   selector: 'app-bootenv-rename',
-  template: '<entity-form [conf]="this"></entity-form>',
+  template: "<entity-form [conf]='this'></entity-form>",
```

Comment:
@dszidi Are there any special reasons for this modification? If no, how about to revert this line.

Reply:
Oops. Seems my regex was flipped. I actually was trying to force single quotes throughout the file. Will fix, nice catch